modules/gapi/include/opencv2/gapi/s11n.hpp

```diff
@@ -17,61 +17,125 @@
 namespace cv {
 namespace gapi {
 
+/**
+* \addtogroup gapi_serialization
+* @{
+*/
+
 namespace detail {
+    /// @private -- Exclude this function from OpenCV documentation
     GAPI_EXPORTS cv::GComputation getGraph(const std::vector<char> &p);
 
+    /// @private -- Exclude this function from OpenCV documentation
     GAPI_EXPORTS cv::GMetaArgs getMetaArgs(const std::vector<char> &p);
 
+    /// @private -- Exclude this function from OpenCV documentation
     GAPI_EXPORTS cv::GRunArgs getRunArgs(const std::vector<char> &p);
 
+    /// @private -- Exclude this function from OpenCV documentation
     GAPI_EXPORTS std::vector<std::string> getVectorOfStrings(const std::vector<char> &p);
 
+    /// @private -- Exclude this function from OpenCV documentation
     template<typename... Types>
     cv::GCompileArgs getCompileArgs(const std::vector<char> &p);
 
+    /// @private -- Exclude this function from OpenCV documentation
     template<typename RMatAdapterType>
     cv::GRunArgs getRunArgsWithRMats(const std::vector<char> &p);
 } // namespace detail
 
+/** @brief This function allows to serialize GComputation.
```

Comment:
Please don't use constructs like this. Just write `Serialize a graph represented by GComputation into an array of bytes`

Reply:
Done